torch/utils/_pytree.py

```diff
@@ -173,21 +173,12 @@ class _SerializeNodeDef(NamedTuple):
 try:
     _optree_version = importlib.metadata.version("optree")
 except importlib.metadata.PackageNotFoundError:
+    # optree was not imported
```

Comment:
`importlib.metadata.version` does not actually import the package. It only looks up the distribution metadata.

Reply:
That's fine, it is looking up if optree was imported

Reply:
Since we do not import `_cxx_pytree.py` here, we still need a version check in `_pytree.py`.

Reply:
OK, it seems that there will be an `ImportError` raised on the first import `_cxx_pytree`.